### ISSUE TITLE
Desired functionalities in CWA vs data/privacy protection

### ISSUE THREAD

**Issue by dsarkar** [corona-warn-app/cwa-wishlist#363]

**Comment by dsarkar** [comment]
@Ein-Tim @ndegendogo  @MikeMcC399 and community ?

**Comment by Ein-Tim** [comment]
IMHO most desired:

- #5 (or something similar which confirms the user that the app works) - informative logging results
- #206 (related: #100, #208, #266) - time/location of risk encounters

**Comment by ndegendogo** [comment]
Yes, I think @Ein-Tim already mentioned the most pressing and desired.
- #5 / #19 / #243 - some way to confirm that everything is setup and working correctly, especially with regard to the BLE beacons. We have seen that some settings (esp. on some Android configurations) can be tricky. But also imagine, my phone fell to the ground and the screen broke - how can I verify that the BLE unit is not also broken?
- #206 and similar (more information / details on risk encounters).
- then there were some ideas to address "superspreader events" and "clusters", like #355 / #279 / #262 / #240 / #213
- and if you are in discussion with Apple & Google, you could also mention #12 (support for Apple Watch)

And, maybe off-topic here, but maybe you can also bring up #109 - "One phone can be used to manage all tests of a family". This feature was very much missed last summer (and I am sure, latest next holiday season this discussion will come up again). It will need some kind of concept for a "local user account management".
